ironfish/src/primitives/transaction.test.perf.ts

```diff
@@ -0,0 +1,127 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+/* eslint-disable no-console */
+import { Asset } from '@ironfish/rust-nodejs'
+import { Assert } from '../assert'
+import { useAccountFixture, useMinerBlockFixture } from '../testUtilities/fixtures'
+import { createRawTransaction } from '../testUtilities/helpers/transaction'
+import { createNodeTest } from '../testUtilities/nodeTest'
+import { BigIntUtils, CurrencyUtils } from '../utils'
+import { BenchUtils } from '../utils/bench'
+import { Account } from '../wallet'
+import { RawTransaction } from './rawTransaction'
+
+type Results = { spends: number; outputs: number; elapsed: number }
+
+describe('Transaction', () => {
+  const nodeTest = createNodeTest()
+
+  const TEST_AMOUNTS = [1, 10, 100]
+
+  it('post', async () => {
+    const { wallet } = nodeTest
+
+    const account = await useAccountFixture(wallet)
+
+    // Generate enough notes for the tests
+    for (let i = 0; i < Math.max(...TEST_AMOUNTS); i++) {
+      const block = await useMinerBlockFixture(
+        nodeTest.chain,
+        undefined,
+        account,
+        nodeTest.wallet,
+      )
+      await expect(nodeTest.chain).toAddBlock(block)
+      await nodeTest.wallet.updateHead()
+    }
+
+    // Spends
+    for (const i of TEST_AMOUNTS) {
+      const results = await runTest(account, i, 1)
+      printResults(results)
+    }
+
+    // Outputs
+    for (const i of TEST_AMOUNTS) {
```

Comment:
the first test is duplicated for outputs. Should we just make TEST_AMMOUNTS a tuple?

Reply:
good call, made it more explicit by making TEST_AMOUNTS an array of { spends, outputs } which is easier to read and removes the duplicate